### PR TITLE
adding progress color

### PR DESCRIPTION
**Pull request by LowellBateman**
@abuecker @johntowell

### CODE REVIEW

File: api/v2/website/schema/website.res.json [hunk 1]
```diff
@@ -39,6 +39,10 @@
         "alternateFontColor": {
           "type": "string",
           "format": "hexcolor"
+        },
+        "progressColor": {
```
Comment (abuecker) [inline api/v2/website/schema/website.res.json]: I see `progressFontColor` above.

Comment (LowellBateman) [inline api/v2/website/schema/website.res.json]: oops, nice catch

Comment (LowellBateman) [inline api/v2/website/schema/website.res.json]: *fixed

**Comment by abuecker** [comment]
:+1:

**Comment by stochastician** [comment]
:+1:

**Pull request merged**